Framework/Geometry/inc/MantidGeometry/Rendering/GeometryHandler.h

```diff
@@ -49,14 +50,17 @@ class MANTID_GEOMETRY_DLL GeometryHandler {
 
 protected:
   IObjComponent *ObjComp; ///< ObjComponent that uses this geometry handler
-  CSGObject *Obj;         ///< Object that uses this geometry handler
+  CSGObject *csgObj;      ///< CSG Object that uses this geometry handler
```

Comment:
Same question as above.

Reply:
I don't remember why in this case, but in the previous pull request #21245 , I was not able to replace usage CSGObject in the geometry handlers by IObject, either because a concrete instance was required or a CSG specific method was called in one of the geometry handlers.